locales/fr-FR.ts

```diff
@@ -171,9 +171,9 @@ export default defineI18nLocale(() => {
         label: 'Étoiles',
       },
       route: {
-        difficulty: 'Difficulté :',
-        length: 'Distance :',
-        duration: 'Durée :',
+        difficulty: 'Difficulté: {difficulty}',
+        length: 'Distance: {length}',
+        duration: 'Durée: {duration}',
```

Comment:
In french space is required before column.

```suggestion
        difficulty: 'Difficulté : {difficulty}',
        length: 'Distance : {length}',
        duration: 'Durée : {duration}',
```

Reply:
ooh really ?? alright